pyccel/ast/class_defs.py

```diff
@@ -159,6 +159,7 @@
             PyccelFunctionDef('copy', func_class = SetCopy),
             PyccelFunctionDef('pop', func_class = SetPop),
             PyccelFunctionDef('remove', func_class = SetRemove),
+            PyccelFunctionDef('discard', func_class = SetDiscard),
```

Comment:
Can we keep the ordering (alphabetical)?

Reply:
Fixed : 55d1560